lib/src/screens/reciters/reciters_screen.dart

```diff
@@ -26,7 +27,7 @@ class RecitersScreen extends ConsumerWidget {
       body: Column(
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
-          const WindowButtons(),
+          if (!UniversalPlatform.isWeb) const WindowButtons(),
```

Comment:
Why dont u just adjust the `WindowButtons` Widget and make it return `Sizedbox.shrink()` if web